src/index.js

```diff
@@ -43,6 +44,15 @@ require('./scripts/loadout/dimLoadoutPopup.directive');
 require('./scripts/loadout/random/dimRandom.controller');
 require('./scripts/compare/dimCompare.directive');
 require('./scripts/compare/dimCompareService.factory');
+require('./scripts/itemDiscuss/dimItemDiscuss.directive');
+require('./scripts/itemDiscuss/dimItemDiscussService.factory');
+require('./scripts/destinyTrackerApi/bulkFetcher.js');
+require('./scripts/destinyTrackerApi/itemListBuilder.js');
+require('./scripts/destinyTrackerApi/itemTransformer.js');
+require('./scripts/destinyTrackerApi/reviewDataCache.js');
+require('./scripts/destinyTrackerApi/reviewsFetcher.js');
+require('./scripts/destinyTrackerApi/reviewSubmitter.js');
+require('./scripts/destinyTrackerApi/trackerErrorHandler.js');
```

Comment:
We can move this stuff around later, but for new functionality we're switching to using imports in specific files rather than just listing them all here. It'll pay dividends later when we can do lazy loading and such. We only just got the capability after moving to webpack, but you can look at the `record-books` module for a good example.